server/resp/src/main/java/org/infinispan/server/resp/logging/Log.java

```diff
@@ -0,0 +1,36 @@
+package org.infinispan.server.resp.logging;
+
+import static org.jboss.logging.Logger.Level.WARN;
+
+import java.nio.CharBuffer;
+
+import org.infinispan.commons.CacheConfigurationException;
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * Log abstraction for the Resp protocol server module. For this module, message ids
+ * ranging from 12001 to 13000 inclusively have been reserved.
+ *
+ * @author William Burns
+ * @since 14.0
+ */
+@MessageLogger(projectCode = "ISPN")
+public interface Log extends BasicLogger {
+   @Message(value = "Cache '%s' has expiration enabled which violates the Resp protocol", id = 12001)
+   CacheConfigurationException invalidExpiration(String cacheName);
+
+   @Message(value = "Cannot enable authentication without an authenticator", id = 12002)
```

Comment:
```suggestion
   @Message(value = "You must configure an authentication mechanism to enable authentication.", id = 12002)
```

Reply:
Does this change sound OK? Seems a little more direct and specific than the original.

Reply:
Seems fine to me, @tristantarrant you mind?

Reply:
what are the authentication mechanisms?

Reply:
there is only one mechanism (the RESP one) so we should not mention them here.